test/unit/Libify.js

```diff
@@ -189,6 +189,23 @@ describe('Libify.buildTransaction', () => {
           name: 'wow',
           value: 'such test'
         }
+      },
+      {
+        id: 10,
```

Comment:
I think id should be 12. See [here](https://www.stellar.org/developers/horizon/reference/resources/operation.html).